src/test/java/com/arpnetworking/kairosdb/integration/AggregationIT.java

```diff
@@ -535,4 +628,18 @@ private String queryWithExpectedCode(
             return body;
         }
     }
+
```

Comment:
Like my above comment: can we refactor the other `queryWithExpectedCode` implementation to be a simple wrapper around this?